runtime/src/message_processor.rs

```diff
@@ -127,15 +157,15 @@ pub fn verify_instruction(
     }
 
     // Credit-only account data may not change.
-    if !is_debitable // line coverage used to get branch coverage
+    if !pre.debitable // line coverage used to get branch coverage
```

Comment:
this clause is really hard to read, right?

Reply:
can we see what it'd look like if we changed the sense of the variable, called it "credit_only"?